librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -494,6 +494,84 @@ commands:
         optional: true
     modes:
       - RZ_OUTPUT_MODE_JSON
+  - name: po
+    summary: Print operation applied on the data
+    subcommands:
+      - name: po2
+        summary: 2-byte endian swap
+        cname: print_operation_2swap
+        args: []
+      - name: po4
+        summary: 4-byte endian swap
+        cname: print_operation_4swap
+        args: []
+      - name: po8
+        summary: 8-byte endian swap
+        cname: print_operation_8swap
+        args: []
+      - name: poa
+        summary: Apply addition
+        cname: print_operation_add
+        args:
+          - name: value
```

Comment:
Aren't these mandatory?

Reply:
No, if they are missing - it takes value from the clipboard (same as with `y` commands, for example). I didn't change this behavior. It's somewhat questionable but that's how it is.

Reply:
Where is this done? I see that `print_operation_transform` accepts `val` and if `val` is not NULL it sets `hex`, but if `val` is not defined it does not do anything. Then `hex` is passed to `rz_core_transform_op`, which does not seem to do anything with the clipboard. Am I missing anything?

Reply:
Ok, they are mandatory now.